boards/PDM/Inc/Io/Io_Aux1Aux2Efuse.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#include <assert.h>
+#include <math.h>
+#include "App_Efuse.h"
+#include "Io_Efuse.h"
+#include "Io_CurrentSense.h"
+
+/**
+ * Initialize the Aux1Aux2 efuse
+ * @param hspi Handle to the SPI peripheral used for the Efuse
+ */
+void Io_Aux1Aux2Efuse_Init(SPI_HandleTypeDef *hspi);
+
+/**
+ * Enable the Aux1 channel output for the Aux1Aux2 efuse.
+ */
+void Io_Aux1Aux2Efuse_Aux1Enable(void);
+
+/**
+ * Disable the Aux1 channel output for the Aux1Aux2 efuse.
+ */
+void Io_Aux1Aux2Efuse_Aux1Disable(void);
+
+/**
+ * Enable the Aux2 channel output for the AuxAux2 efuse
+ */
+void Io_Aux1Aux2Efuse_Aux2Enable(void);
+
+/**
+ * Disable the Aux 2 channel output for the AuxAux2 efuse
+ */
+void Io_Aux1Aux2Efuse_Aux1Disable(void);
+
+/**
+ * Get the value of the Aux1Aux2 efuse's Status Register
+ * @param channel_status Pointer to the Aux1 and Aux2 channel's status
+ * @return EXIT_CODE_OK if the read was successful,
+ *         else EXIT_CODE_TIMEOUT if the SPI read timed-out
+ */
+ExitCode Io_Aux1Aux2Efuse_GetStatus(enum Efuse_Status *status);
+
+/**
+ * Get the value of the Aux 1 Fault Register over SPI
+ * @note Reading from the fault register reset's non-latchable faults
+ * @param fault_status Pointer to the Aux 1 fault status
+ * @return EXIT_CODE_OK if the read was successful,
+ *         else EXIT_CODE_TIMEOUT if the SPI read timed-out
+ */
+ExitCode Io_Aux1Aux2Efuse_GetAux1Faults(enum Efuse_Fault *fault);
+
+/**
+ * Get the value of the Aux 2 Fault Register over SPI
+ * @note Reading from the fault register reset's non-latchable faults
+ * @param fault Pointer to the Aux 2 fault status
+ * @return EXIT_CODE_OK if the read was successful,
+ *         else EXIT_CODE_TIMEOUT if the SPI read timed-out
+ */
+ExitCode Io_Aux1Aux2Efuse_GetAux2Faults(enum Efuse_Fault *fault);
+
+/**
+ * Check if the Efuse's Aux1 or Aux2 channel on the Aux1Aux2 efuse enters fault
+ * mode
+ * @return true if the Efuse's Aux1 or Aux2 channel enters fault mode, else
+ * false if both channels do not enter fault mode
+ */
+bool Io_Aux1Aux2Efuse_IsInFaultMode(void);
+
+/**
+ * Check if the Aux1/Aux2 Efuse enters fail-safe mode
+ * @param e_fuse Pointer to the Aux1Aux2 Efuse structure
```

Comment:
Make sure these are removed for the functions below eh. Possibly a leftover moment

Reply:
yar